asyn/drvAsynFTDI/drvAsynFTDIPort.cpp

```diff
@@ -511,8 +517,17 @@ drvAsynFTDIPortConfigure(const char *portName,
                          int latency,
                          unsigned int priority,
                          int noAutoConnect,
-                         int noProcessEos)
+                         int flags)
 {
+
+    int noProcessEos = ((flags & NO_PROC_EOS_BIT)==NO_PROC_EOS_BIT)?1:0;
+    int SPI = ((flags & UART_SPI_BIT)==UART_SPI_BIT)?1:0;
```

Comment:
I understand repurposing the `noProcessEos` argument makes it easy to implement this feature, but I worry that this might be confusing for the user since other asynDrivers do not do that (AFAIK). However, I'd defer to @MarkRivers on this point. @jtagger , would it be possible to set the SPI/UART operation mode through the existing `setOption`/`getOption` interface? Or does it have to be set at creation time?

Reply:
Also, instead of `int SPI`, could you call this local variable `int mode` or something similar? Further below when you do `ftdi->FTDImode = SPI` I thought for a moment that you were defaulting to SPI mode.